app/services/rate_sleeper.rb

```diff
@@ -0,0 +1,34 @@
+class RateSleeper
+  attr_reader :jobs_per_second
```

Comment:
super pedantic so ignore if you fancy, but we could set the `attr_reader` in `private` since it is only used internally.

Reply:
I expect nothing less from you... Fixed in 37a16a9